src/covid19model/models/models.py

```diff
@@ -138,9 +138,9 @@ def integrate(t, S, E, I, A, M, ER, C, C_icurec, ICU, R, D, H_in, H_out, H_tot,
 
         # Compute the  rates of change in every population compartment
         # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-        # beta => beta*(1-alpha)+1.6*beta*alpha
-        dS  = - beta*s*np.matmul(Nc,((I+A)/T))*S + zeta*R - v*e*S
-        dE  = beta*s*np.matmul(Nc,((I+A)/T))*S - E/sigma - v*e*E
+        # beta => (beta*(1-alpha)+1.6*beta*alpha)
+        dS  = - (beta*(1-alpha)+1.6*beta*alpha)*s*np.matmul(Nc,((I+A)/T))*S + zeta*R - v*e*S
+        dE  = (beta*(1-alpha)+1.6*beta*alpha)*s*np.matmul(Nc,((I+A)/T))*S - E/sigma - v*e*E
```

Comment:
@JennaVergeynst Just a preference for the sake of code clarity, can you recompute `beta` on one line and then use the new `beta` in the equations for dS and dE? Like this,
```
# Compute weighted average beta
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
beta =  beta*(1-alpha)+1.6*beta*alpha

# Compute the  rates of change in every population compartment
# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
dS  = - beta*s*np.matmul(Nc,((I+A)/T))*S + zeta*R - v*e*S
```

Reply:
I thought of it, but I'm doubting, because at one hand it would be better to give another name to the recomputed beta then, but at the other hand, it's better not to rename beta for overall consistency...

Reply:
ah no, it's no problem actually, ok, I will change it!